charts/library/common-test/tests/volsync/replication_src_spec_test.yaml

```diff
@@ -437,7 +437,7 @@ tests:
               src:
                 enabled: true
       credentials:
-        mys3:
+        Mys3:
```

Comment:
If you want to test this, please write a separate test and dont willy-nilly sneak it into an existing test with a different description